apply linter to rpc test files

rohan-varma opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#32659 apply linter to rpc test files**

Applies linter to RPC test files so that we can use linter shortcuts
without getting unnecessary changes to the whole file.

Differential Revision: [D19584742](https://our.internmc.facebook.com/intern/diff/D19584742/)

rohan-varma committed:
apply linter to rpc test files

Applies linter to RPC test files so that we can use linter shortcuts
without getting unnecessary changes to the whole file.

Differential Revision: [D19584742](https://our.internmc.facebook.com/intern/diff/D19584742/)

[ghstack-poisoned]

xush6528 commented:
Thanks! want this for a long time.

xush6528 commented on torch/testing/_internal/distributed/rpc/dist_optimizer_test.py:
```diff
@@ -1,15 +1,17 @@
 from __future__ import absolute_import, division, print_function, unicode_literals
```
It's odd for a py3 only module to have this line.

rohan-varma replied:
yep, I think all RPC/dist autograd code was py 3 only, but some parts of the PT codebase are going to support py 2 for a bit longer. I think this should we removed when we officially drop our python 2 support.

rohan-varma committed:
Update on "apply linter to rpc test files"

backward_node_failure by adding more error regexes

Applies linter to RPC test files so that we can use linter shortcuts
without getting unnecessary changes to the whole file.

Differential Revision: [D19584742](https://our.internmc.facebook.com/intern/diff/D19584742/)

[ghstack-poisoned]

kostmo commented:
## :pill: CircleCI build failures summary and remediations
As of commit 35a9e090:


**None of the build failures appear to be your fault**.

* **1/1** [broken upstream](https://dr.pytorch.org/master-timeline.html?count=1&sha1=f6c46df856d0588360db5b807960d1fc5e888c36&should_suppress_scheduled_builds=true&should_suppress_fully_successful_columns=true&max_columns_suppress_successful=35) at merge base f6c46df8 since Jan 27
  <details>
  <summary>
  Please rebase on the <code>viable/strict</code> branch (expand for instructions)
  </summary>
  
  If your commit is newer than `viable/strict`, you can try basing on an older, stable commit:
  ```
  git fetch origin viable/strict
  git rebase --onto viable/strict $(git merge-base origin/master HEAD)
  ```
  If your commit is older than `viable/strict`:
  ```
  git fetch origin viable/strict
  git rebase viable/strict
  ```
  Check out the [recency history](https://dr.pytorch.org/master-viable-commits.html) of this "viable master" tracking branch.
  
  </details>
  


## Detailed failure analysis

One may explore the probable reasons each build failed interactively [on the Dr. CI website](https://dr.pytorch.org/commit-details.html?sha1=35a9e090bc8c5d64972bd6cf5e3ed706c4ed6e62).

### :construction: 1 upstream failure recognized by patterns:
These builds matched patterns, but were probably **caused by upstream breakages**:
* [pytorch_macos_10_13_py3_test](https://dr.pytorch.org/build-details.html?build_id=91679355) from Jan 27 until Jan 27



---

<details><summary>This comment was automatically generated by <a href="https://github.com/kostmo/circleci-failure-tracker/tree/master/docs/from-pull-request-comment">Dr. CI</a> (expand for details).</summary>Follow <a href="https://dr.pytorch.org/admin/comments-opt-out.html">this link to opt-out</a> of these comments for your Pull Requests.

Please report bugs/suggestions on the <a href="https://github.com/kostmo/circleci-failure-tracker/issues">GitHub issue tracker</a>.
</details>

rohan-varma committed:
Update on "apply linter to rpc test files"

backward_node_failure by adding more error regexes
backward_node_failure by adding more error regexes

Applies linter to RPC test files so that we can use linter shortcuts
without getting unnecessary changes to the whole file.

Differential Revision: [D19584742](https://our.internmc.facebook.com/intern/diff/D19584742/)

[ghstack-poisoned]
